addons/dragging/XEH_clientInit.sqf

```diff
@@ -22,4 +22,23 @@ if (isNil "ACE_maxWeightCarry") then {
 // handle waking up dragged unit and falling unconscious while dragging
 ["medical_onUnconscious", DFUNC(handleUnconscious)] call EFUNC(common,addEventhandler);
 
+[
+    "ACE 3 Common",
+    localize LSTRING(Drop),
+    {
+        // End Dragging
+        if !(isNull (ACE_player getVariable [QGVAR(draggedObject),objNull])) exitWith {
+            [ACE_player, ACE_player getVariable [QGVAR(draggedObject),objNull]] call FUNC(dropObject);
+            true
+        };
+        // End Carry
+        if (!isNull (ACE_player getVariable [QGVAR(carriedObject),objNull])) exitWith {
+            [ACE_player, ACE_player getVariable [QGVAR(carriedObject),objNull]] call FUNC(dropObject_carry)
+            true
+        };
+        false
+    },
+    { false },
+    [0, [false, false, false]]
```

Comment:
Default is `[-1, [false, false, false]]` anyways. Having this there is imo more confusing, since you don't omit setting the default itself, making it feel like this is actually a value to be considered.